packages/@vue/cli/lib/promptModules/e2e.js

```diff
@@ -25,6 +25,11 @@ module.exports = cli => {
         name: 'Nightwatch (WebDriver-based)',
         value: 'nightwatch',
         short: 'Nightwatch'
+      },
+      {
+        name: 'WebdriverIO (WebDriver/DevTools based)',
+        value: 'webdriverio',
+        short: 'WebdriverIO'
```

Comment:
Should chnage line 39 from `when: answers => answers.e2e === 'nightwatch'` to `when: answers => answers.e2e === 'nightwatch' || answers.e2e === 'webdriverio'` (I can't directly add review comment on that line in the GitHub web ui)